package.json

```diff
@@ -6,7 +6,10 @@
     "type": "git",
     "url": "https://github.com/latticexyz/mud.git"
   },
-  "private": true,
+  "exports": {
+    "./cli": "./packages/cli/dist/index.js"
+  },
+  "bin": "./packages/cli/dist/mud.js",
```

Comment:
For now we only release the `cli` package under the global `mud` package, but by adding more entry points here (and adding exceptions for the respective packages to `.npmignore`) we can easily start publishing all packages under `mud/<package>`!